frontend/src/pages/staff-dashboard/TestResult/TestResultsManagement.jsx

```
@@ -130,13 +171,46 @@ const TestResultsByService = () => {
+    // Filter by search term (name or phone)
+    if (searchTerm) {
+      const term = searchTerm.toLowerCase()
+      filteredData = filteredData.filter(
+        (item) => item.phone.toLowerCase().includes(term) || item.customerName.toLowerCase().includes(term)
+      )
+    }
+
+    // Filter by date range
+    if (selectedDateRange && selectedDateRange[0] && selectedDateRange[1]) {
+      filteredData = filteredData.filter((item) => {
+        const itemDate = new Date(item.testDate)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)